etc/ci/performance/test_all.sh

```diff
@@ -8,19 +8,29 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+while (( "${#}" ))
+do
 case "${1}" in
   --servo)
     engine="--engine servo"
     ;;
   --gecko)
     engine="--engine gecko"
     ;;
+  --local)
```

Comment:
Since most people trying this out will be running it locally, would you mind inverting the flag? Let's NOT submit to perfherder by default, and trigger the `submit_to_perfherder.py` only if we specify `--upload-result`.